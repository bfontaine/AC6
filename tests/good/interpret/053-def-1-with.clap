def f (x) = y with y = 42
val q = f (1)

