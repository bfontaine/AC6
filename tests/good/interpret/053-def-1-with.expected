:- f = <code>
:- y = <code>
:- q = <code>
